COMPILING.md

```diff
@@ -1,30 +1,31 @@
+# Compiling Jamulus
+
 ## Points to note
 
-- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the binaries generated by the autobuild process in the Jamulus repository.
+- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the [binaries generated by the autobuild process](https://github.com/jamulussoftware/jamulus/releases/latest) in the Jamulus repository.
 - For unattended installs, see the contributed [installation scripts](https://github.com/jamulussoftware/installscripts)
+- The [Autobuild scripts](https://github.com/jamulussoftware/jamulus/tree/master/.github/autobuild) although only optimized to be used by the CI, might help you understand the process of setting up a build environment
 - There are reports from people who successfully compile and run Jamulus on BSDs.
 - Android and iOS are not officially supported.
```

Comment:
```suggestion
- Android and iOS are not officially supported.  However, [binaries are generated by the autobuild process](https://github.com/jamulussoftware/jamulus/releases/latest) in the Jamulus repository.
```

Reply:
If this is going to be update, it's probably also worth mentioning the binary for Jack on Windows.

Reply:
So you'd consider JACK not officially supported?